internal/log/log.go

```diff
@@ -167,8 +167,17 @@ func (l *Logger) LogOp(op func() error, format string, a ...interface{}) error {
 
 	l.logStart(format, a...)
 	if err := op(); err != nil {
-		l.logFail("%s: %v", fmt.Sprintf(format, a...), err)
-		return err
+		// catching CheckIfUserExists - exit status 1 and Stderr Empty
+		//
+		// How do I catch if Stderr in err is empty ?
+		if strings.ContainsAny(err.Error(), "exit status 1") && strings.ContainsAny(err.Error(), "Stderr \"\" ") {
```

Comment:
There could be other situations in Ignition where a command it's calling would exit with status 1 and no logs on stderr aside from the `CheckIfUserExists` use case.

Instead of modifying `LogOp` to work around this, I think I'd rather have `CheckIfUserExists` not use `LogCmd`. Even aside from the logging level, a user seeing that calling `id` failed in the logs could be confusing.